extensions/adobe/experience/campaign/experienceevent.example.click.json

```diff
@@ -0,0 +1,100 @@
+{
+  "xdm:timestamp": 1522321544448,
```

Comment:
Please take a look at the definition of `xdm:timestamp`, it's a `date-time`-formatted `string`

Reply:
:+1: